substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -1756,18 +1761,23 @@ def build_and_test_java_agent_image(native_image, args):
             # Note: we are not using MX here to avoid polluting the suite.py and requiring extra build flags
             mx.log("Building agent jars from " + test_classpath)
             agents = []
-            for i in range(1, 2):
+            for i in range(1, 3):
```

Comment:
I don't see the `agent3` in the PR? Is it committed?

Reply:
python's `range(x, y)` includes x but excludes y.